srcpkgs/blender/template

```diff
@@ -21,7 +21,7 @@ patch_args="-Np1"
 pycompile_version="$py3_ver"
 pycompile_dirs="/usr/share/blender/$version/scripts"
 
-only_for_archs="x86_64 x86_64-musl i686 i686-musl ppc64le ppc64le-musl ppc64-musl"
+archs="x86_64* i686* ppc64le ppc64le-musl ppc64-musl"
```

Comment:
@q66 can this be reduced to `x86_64* i686* ppc64*`?

Reply:
Go ahead.